plookup/README.md

```diff
@@ -0,0 +1,4 @@
+# Plookup on Plonky
+
+### Notes
+Uses const generics and thus requires Rust nightly.
```

Comment:
Seems fine for now but let's try to avoid other nightly stuff if we can, so that if it ends up blocking some projects from using Plonky, we can switch back to stable without too much work.

Reply:
Yes I used const generics mostly out of curiosity and because I think they fit well for this use case. But it should be pretty easy to get the same behavior with `Vec`s if we need to run on stable.